### DISCUSSION TITLE
Video loading very slow in Ente Photos (desktop and iOS apps)

### DISCUSSION THREAD

**Discussion by Polygon1556** [ente-io/ente#3566] [General]
I've noticed that video loading in Ente Photos is extremely slow on both the desktop application and the iOS mobile app. It takes several minutes for a video to load, making the feature practically unusable for me. I'm not sure if the delay is related to decryption, but the issue persists across both platforms.

Desktop: Debian 12
Mobile: iOS 18
Internet Speed: 200 mbps

**Comment by pavel-z-ostravy** [comment]
Same issue on 500mbps internet. Tested also Proton Drive and Mega, no such issues there. But Still rooting for Ente though :-) good luck fixing this
